src/main/java/core/basesyntax/bookstore/repository/BookRepository.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.bookstore.repository;
+
+import core.basesyntax.bookstore.model.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
not a mistake, but  FYI Repository is redundant when we extend JpaRepository interface